README.md

```diff
@@ -1,15 +1,19 @@
 # Cloud Orchestrator
 
-[Cloud Orchestrator](https://cloud.airgap.host) is a Drupal cloud management distribution tailored for cloud administrators and operators.
+[Cloud Orchestrator](https://cloud.airgap.host) is a Drupal cloud management
+distribution tailored for cloud administrators and operators.
 
-The distribution manages resources in public and private clouds. Version 5.x includes support for Amazon EC2 and Amazon VPC/Networking, Kubernetes, OpenStack, and Terraform.
+The distribution manages resources in public and private clouds. Version 6.x
+includes support for Amazon EC2 and Amazon VPC/Networking, Kubernetes,
+OpenStack, and Terraform.
```

Comment:
Let's add VMware too.

Reply:
@baldwinlouie Thank you for your review.  I added `VMware`.  Can you please check the patch again?